api/integrations/lead_tracking/hubspot/lead_tracker.py

```diff
@@ -0,0 +1,78 @@
+import logging
+
+from django.conf import settings
+
+from integrations.lead_tracking.lead_tracking import LeadTracker
+from organisations.models import HubspotOrganisation, Organisation
+from users.models import FFAdminUser
+
+from .client import HubspotClient
+
+logger = logging.getLogger(__name__)
+
+try:
+    import re2 as re
+
+    logger.info("Using re2 library for regex.")
+except ImportError:
+    logger.warning("Unable to import re2. Falling back to re.")
+    import re
```

Comment:
This import logic was to mirror the functionality of the existing pipedrive code which had this way of importing `re`.